shared/VCMetadata.ts

```diff
@@ -133,7 +133,7 @@ export const getVCMetadata = (context: object, keyType: string) => {
   });
 };
 
-const getDisplayId = (
+const getMosipIndividualId = (
```

Comment:
Remove logic for mDoc or policy number.
Add check for isMosipVC() then only return uin/vid else empty string